packages/dataviews/src/dataviews-layouts/picker-table/style.scss

```diff
@@ -0,0 +1,45 @@
+// Picker-specific table styles
+// The table picker mainly reuses the regular table styles from ../table/style.scss
+// Add any picker-specific overrides here if needed in the future.
+
+.dataviews-view-table {
+	tbody:focus-visible {
+		// Only show one focus outline at a time. When focus is on a child element,
+		// hide the table's own focus outline.
+		&[aria-activedescendant] {
+			outline: none;
+		}
+
+		[data-active-item="true"] {
+			outline: 2px solid var(--wp-admin-theme-color);
+		}
+	}
+
+	.dataviews-selection-checkbox {
+		// Added specificity to override table styles.
+		.components-checkbox-control__input.components-checkbox-control__input {
+			// When the dataview is used as a picker, ensure the checkbox can't be clicked on.
+			// Only the row itself is clickable.
+			pointer-events: none;
```

Comment:
`specs/site-editor/page-list.spec.js` E2E tests fail locally for me too because of this line. 

I think because the checkbox has `pointer-events: none`, so it can't be clicked. The row is clickable, but the test clicks the checkbox.

Wrapping `.dataviews-picker-wrapper ` around these styles appears to work, but not sure if that's a decent workaround for now 🤔

Reply:
Fixed! I added a classname specific to the picker.